cmd/podman/images.go

```diff
@@ -280,7 +280,9 @@ func getImagesTemplateOutput(ctx context.Context, runtime *libpod.Runtime, image
 		if !opts.noTrunc {
 			imageID = shortID(img.ID())
 		}
+
 		// get all specified repo:tag pairs and print them separately
+	outer:
```

Comment:
interesting, I'd this on the line below, but gofmt automagically moved it for me apparently.